cert-manager/namespaces/cert-manager/Deployment-cert-manager.yaml

```diff
@@ -1,61 +1,47 @@
-# Source: cert-manager/templates/deployment.yaml
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: cert-manager
-  namespace: "cert-manager"
   labels:
     app: cert-manager
-    app.kubernetes.io/name: cert-manager
+    app.kubernetes.io/component: controller
     app.kubernetes.io/instance: cert-manager
-    app.kubernetes.io/managed-by: Tiller
-    helm.sh/chart: cert-manager-v0.13.1
+    app.kubernetes.io/name: cert-manager
+  name: cert-manager
+  namespace: cert-manager
 spec:
   replicas: 1
   selector:
     matchLabels:
-      app: cert-manager
-      app.kubernetes.io/name: cert-manager
+      app.kubernetes.io/component: controller
       app.kubernetes.io/instance: cert-manager
-      app.kubernetes.io/managed-by: Tiller
+      app.kubernetes.io/name: cert-manager
```

Comment:
We'll need to delete and re-create the deployment resources due to this selector change, but that shouldn't be an issue :)